cluster/gce/util.sh

```diff
@@ -745,16 +745,17 @@ function check-cluster() {
       if [[ ${elapsed} -gt ${KUBE_CLUSTER_INITIALIZATION_TIMEOUT} ]]; then
           echo -e "${color_red}Cluster failed to initialize within ${KUBE_CLUSTER_INITIALIZATION_TIMEOUT} seconds.${color_norm}" >&2
           if [[ ${KUBE_TEST_DEBUG-} =~ ^[yY]$ ]]; then
-            local tmp_log="$(mktemp)"
-            local file
-            for file in /var/log/startupscript.log /var/log/kube-apiserver.log; do
-              echo "${MASTER_NAME}:${file} contents:"
-              if gcloud compute copy-files --project "${PROJECT}" \
-                --zone "${ZONE}" "${MASTER_NAME}:${file}" "${tmp_log}"; then
-                cat "${tmp_log}"
-              fi
+            local logdir=/var/log
+            local basename
+            for basename in startupscript kube-apiserver; do
+              local src="${logdir}/${basename}.log"
+              echo "Preserving ${MASTER_NAME}:${src}"
+              gcloud compute copy-files \
+                -project "${PROJECT}" --zone "${ZONE}" \
+                "${MASTER_NAME}:${src}" \
+                "${E2E_REPORT_DIR}/${basename}.${MASTER_NAME}.log.$(date +'%Y%m%d-%H%M%S')" \
```

Comment:
also, `E2E_REPORT_DIR` isn't necessarily set. So either use a default here, or remove `KUBE_TEST_DEBUG` and case on `E2E_REPORT_DIR` being set.

(probably we should *always* have `E2E_REPORT_DIR` set to something, even for local test runs. but that's even more refactoring...)